utils/endpoint.py

```diff
@@ -3,11 +3,19 @@
 
 from utils import fixedlist
 
+import rethinkdb as r
+import json
+
 
 class Endpoint(ABC):
     def __init__(self):
         self.avg_generation_times = fixedlist.FixedList(20)
         self.hits = 0
+        self.config = json.load(open('config.json'))
+        self.RDB_ADDRESS = self.config['rdb_address']
+        self.RDB_PORT = self.config['rdb_port']
+        self.RDB_DB = self.config['rdb_db']
+        self.rdb = r.connect(self.RDB_ADDRESS, self.RDB_PORT)

```

Comment:
`db=RDB_DB`, this removes the need to do `r.db(RDB_DB).table(...)` on each call.